app/views/admin/comments/_comment.html.erb

```diff
@@ -0,0 +1,48 @@
+<div>
```

Comment:
This external div is sort of a preparation for next PR as in that PR we will add more child items to this `div`